src/main/java/seedu/address/model/person/PersonContainsAllTagsPredicate.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.person;
+
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Tests that a {@code Person}'s {@code Tags} matches all the keywords given.
```

Comment:
Minor nit about phrasing, as a Person with tags `[A], [B], [C]` will pass with keywords `A, B`.

Same applies for TCATP below.

```suggestion
 * Tests that a {@code Person} has {@code Tags} that matches all the keywords given.
```